src/main/java/com/bookstore/OnlineBookstoreApplication.java

```diff
@@ -2,10 +2,14 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = {"com.bookstore"})
+@EnableJpaRepositories(basePackages = "com.bookstore.repository")
+@EntityScan(basePackages = "entity")
```

Comment:
Do you really need all these annotations with arguments? 
```
@SpringBootApplication(scanBasePackages = {"com.bookstore"})
@EnableJpaRepositories(basePackages = "com.bookstore.repository")
@EntityScan(basePackages = "entity")
```

You can leave only `@SpringBootApplication`